### PR TITLE
Prevent Apache to manage the docroot for foreman-ssl

### PR DESCRIPTION
**Pull request by peimanja**

### CODE REVIEW

**Comment by ekohl** [comment]
Looks like our tests should be adjusted for this as well.

File: spec/classes/foreman_config_passenger_spec.rb [hunk 1]
```diff
@@ -81,10 +81,6 @@
           http_dir = '/etc/apache2'
         end
 
-        it 'should contain the docroot' do
-          should contain_file("#{params[:app_root]}/public")
```
Comment (ekohl) [inline spec/classes/foreman_config_passenger_spec.rb]: Would you mind changing this to ``should_not```? That would be the proper test now.

File: spec/classes/foreman_config_passenger_spec.rb [hunk 2]
```diff
@@ -81,10 +81,10 @@
...
+        it 'should not contain the docroot' do
+          should_not contain_file("#{params[:app_root]}/public")
         end
-
+
```
Comment (mmoll) [inline spec/classes/foreman_config_passenger_spec.rb]: trailing whitespace here

**Comment by mmoll** [comment]
While we can do that in merge, removing the trailing whitespace and squashing together the commits into one would be nice. 👍 otherwise.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**